libraries/botbuilder-dialogs-adaptive/src/input/choiceInput.ts

```diff
@@ -129,6 +132,28 @@ export class ChoiceInput extends InputDialog implements ChoiceInputConfiguration
         }
     }
 
+    /**
+     * {@inheritDoc InputDialog.trackGeneratorResultEvent}
+     */
+    protected trackGeneratorResultEvent(
+        dc: DialogContext,
+        activityTemplate: TemplateInterface<Partial<Activity>, DialogStateManager>,
+        msg: Partial<Activity>
+    ): void {
+        const options = dc.state.getValue(ChoiceInput.OPTIONS_PROPERTY);
+        const properties: { [key: string]: string } = {
+            template: JSON.stringify(activityTemplate),
```

Comment:
Is JSON.stringify needed for these?

Reply:
I don't think it is needed, the properties in [trackGeneratorResultEvent ](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-dialogs-adaptive/src/input/inputDialog.ts#:~:text=%7D-,this.telemetryClient.trackEvent(%7B,%7D%2C,-%7D)%3B)function in base class are not using JSON.stringify. 

I tested without stringifying for these properties and updated the unit test and it passed.

thanks, good catch! making these changes.